inc/class-s3-uploads-image-editor-imagick.php

```diff
@@ -47,40 +89,8 @@ protected function _save( $image, $filename = null, $mime_type = null ) {
 		);
 	}
 
-	public function load() {
-		$result = parent::load();
-
-		// `load` can call pdf_setup() which has to copy the file to a temp local copy.
-		// In this event we want to clean it up once `load` has been completed.
-		if ( $this->temp_file_to_cleanup ) {
-			unlink( $this->temp_file_to_cleanup );
-			$this->temp_file_to_cleanup = null;
-		}
-		return $result;
-	}
-
-	/**
-	 * Sets up Imagick for PDF processing.
-	 * Increases rendering DPI and only loads first page.
-	 *
-	 * @since 4.7.0
-	 *
-	 * @return string|WP_Error File to load or WP_Error on failure.
-	 */
-	protected function pdf_setup() {
```

Comment:
Why remove this?

Reply:
PDF handling is now called in `load`, and given now we are preparing the the `load` with a local disk file, this workaround is not longer needed.